blocks/lede/lede.css

```diff
@@ -0,0 +1,17 @@
+.lede p {
```

Comment:
do these need to be on the paragraphs specifically? can we add the `util` type class via javascript instead of restyling?

Reply:
> Let's try to use the utility type classes, or make a new blockquote style utility and consolidate that with both blocks.

I've added a commit that adds a utility class that's now used by both lede and quote. The blockquote was using the same font sizes, just a different weight. The lede now actually does not need any unique CSS, but it'll cause errors if the CSS file is missing, so I left an empty CSS files. That's something to look at next week.